ctapipe/reco/ImPACT.py

```diff
@@ -377,33 +394,33 @@ def get_likelihood(self, source_x, source_y, core_x, core_y,
         # everything in the correct units when loading in the class
         # and ignore them from then on
 
-        zenith = 90 * u.deg - self.array_direction[0]
-        azimuth = self.array_direction[1]
-        # Geometrically calculate the depth of maximum given this test position
+        zenith = 90*u.deg - self.array_direction.alt
+        azimuth = self.array_direction.az
 
+        # Geometrically calculate the depth of maximum given this test position
         if self.fit_xmax:
             x_max = self.get_shower_max(source_x, source_y,
                                         core_x, core_y,
                                         zenith.to(u.rad).value) * x_max_scale
-
             # Calculate expected Xmax given this energy
             x_max_exp = 300 + 93 * np.log10(energy)
```

Comment:
In this case the expected Xmax is based on what I use in the template generation, will add a function and document.